### PR TITLE
[Imporve][Seatunnel-code][Seatunnel-flink-starter] customize operator parallel for flink and spark

### LINKED ISSUE #2925: [Feature][DISCUSS] operator support customize some feature configurations,like parallel
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Sometimes transform need more parallel, so i think operator support customize some feature configurations.

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by liugddx**
close #2925

## Purpose of this pull request

customize operator parallel

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by hailin0** [comment]
great

File: seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/resources/examples/fake_to_console.conf [hunk 1]
```diff
@@ -53,8 +53,10 @@ transform {
 }
 
 sink {
-  Console {}
+  Console {
+    execution.parallelism = 3
```
Comment (EricJoy2048) [inline seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/resources/examples/fake_to_console.conf]: I suggest we use the unified parameters `parallelism `

File: seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/resources/examples/fake_to_console.conf [hunk 2]
```diff
@@ -28,33 +28,35 @@ env {
...
   # This is a example source plugin **only for test and demonstrate the feature source plugin**
-    FakeSource {
-      result_table_name = "fake"
-      row.num = 16
-      schema = {
-        fields {
-          name = "string"
-          age = "int"
-        }
+  FakeSource {
```
Comment (Hisoka-X) [inline seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/resources/examples/fake_to_console.conf:31]: Please add parallelism config in FakeSource. Also please update doc

**Review by EricJoy2048 — DISMISSED** [review]
+1

File: seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java [hunk 3]
```diff
@@ -76,7 +78,11 @@ public List<DataStream<Row>> execute(List<DataStream<Row>> upstreamDataStreams)
             SeaTunnelSink<SeaTunnelRow, Serializable, Serializable, Serializable> seaTunnelSink = plugins.get(i);
             DataStream<Row> stream = fromSourceTable(sinkConfig).orElse(input);
             seaTunnelSink.setTypeInfo((SeaTunnelRowType) TypeConverterUtils.convert(stream.getType()));
-            stream.sinkTo(new FlinkSink<>(seaTunnelSink)).name(seaTunnelSink.getPluginName());
+            DataStreamSink<Row> dataStreamSink = stream.sinkTo(new FlinkSink<>(seaTunnelSink)).name(seaTunnelSink.getPluginName());
```
Comment (hailin0) [inline seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java:81]: How to support spark?

**Comment by liugddx** [comment]
@Hisoka-X PTALthanks

**Comment by liugddx** [comment]
@ashulin help to review thanks.

**Comment by Hisoka-X** [comment]
Can you add spark for support this parameter too? Because you put `parallelism` into common-options, but spark not support.

**Comment by liugddx** [comment]
> Can you add spark for support this parameter too? Because you put `parallelism` into common-options, but spark not support.

Done. @EricJoy2048 @Hisoka-X

**Review by Hisoka-X — APPROVED** [review]
LGTM

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**